### PR TITLE
Use SolarEdge Environmental Figures

### PR DESCRIPTION
**Pull request by AndrewJ41**
- Fixed a bug where battery charge power was getting added into total DC Output, this should only be the case if the battery is in a discharge state
- Removed Environmental Impact calculation and replaced with SolarEdge stats
- Added indicator to show if battery is in charge, discharge or idle state.
- Made all optional or configurable elements xbar variables.

### CODE REVIEW

File: Environment/solaredge.5m.py [hunk 1]
```diff
@@ -10,6 +10,9 @@
 # <xbar.var>string(SITE_ID): Please provide your SolarEdge installation SiteId</xbar.var>
 # <xbar.var>string(API_KEY): Please provide your SolarEdge installation API Key (https://www.solaredge.com/node/88689)</xbar.var>
 # <xbar.var>string(BATTERY_PRESENT): Does your SolarEdge installation include a battery? Y/N</xbar.var>
+# <xbar.var>string(SYSTEM_WATTAGE): Total panel DC watt capacity</xbar.var>
+# <xbar.var>string(SHOW_ENVIRONMENTAL_BENEFIT): Do you want to show your environmental benfit in the dropdown? Y/N (Default Y)</xbar.var>
```
Comment (sprak3000) [inline Environment/solaredge.5m.py:14]: Why not a boolean variable here?

File: Environment/solaredge.5m.py [hunk 2]
```diff
@@ -10,6 +10,9 @@
...
+# <xbar.var>string(ENVIRONMENTAL_BENEFIT_UNIT): Units to display benefit in? Imperial/Metrics (Default Metrics)</xbar.var>
```
Comment (sprak3000) [inline Environment/solaredge.5m.py:15]:
```suggestion
#  <xbar.var>select(ENVIRONMENTAL_BENEFIT_UNIT="Metrics"): Which style to use. [Imperial, Metrics]</xbar.var>
```

**Review by sprak3000 — COMMENTED** [review]
@AndrewJ41 

Looks fine; just some suggestions around the variables.

File: Environment/solaredge.5m.py (hunk 1 above)
Comment (AndrewJ41) [reply Environment/solaredge.5m.py:14]: More readability than anything else - can change to bool if you think it helps.

Comment (sprak3000) [reply Environment/solaredge.5m.py:14]: 🤷🏽 I'm not a python expert; if you feel this is more readable, that works for me.

**Pull request merged**